include/boost/json/detail/basic_parser.hpp

```diff
@@ -105,6 +107,7 @@ class basic_parser
     bool more_; // false for final buffer
     bool complete_ = false; // true on complete parse
     bool is_key_;
+    parse_options opt_ = {false, false};
```

Comment:
But... `parse_options` already has these set to false in the default ctor?

Reply:
This is a bit of a dilemma -- while `parser` does this, the default constructor for `basic_parser` will not. We can change it though